### DISCUSSION TITLE
Packing Entropy server file format (Read if you do not see the results)

### DISCUSSION THREAD

**Discussion by cgseitz** [Pranavkhade/PACKMAN#45] [Q&A]
Hello,

I am trying to calculate the entropy of a pdb structure. I uploaded it to the Packing Entropy server, and it said

`
Job Status: SUCCESS
`
`
ERROR:root:Please check the parameters.
`

I opened the `
log.txt` which said the same thing, and the `
output.txt` did not contain anything. I did not change any of the default parameters, and did not use any chains in the entropy calculation. Considering the error message says to check the parameters, which are all the default parameters, do you know what may be wrong? I have attached my pdb file as a zip file. Thanks!

Best,
Christian
[subset.zip](https://github.com/Pranavkhade/PACKMAN/files/11143858/subset.zip)

**Comment by Pranavkhade** [comment]
Thank you @cgseitz for reporting this issue. I will contact the university IT department (@njbooher) about the job not showing the error message. However, after closely observing the file you provided, I can see that the PDB file is most likely generated using some other software, and is not a crystal structure from experiments (RCSB PDB database). I will describe what is happening with your file so that you will have a clearer picture and it will help you to decide your next steps.

**What exactly is happening?**
The 'element' column in your file is empty; probably because the software you are using doesn't provide that section/misplaces that section in such a way that it doesn't follow the PDB file norms. Since entropy calculation depends on the type of atom present at a particular position (to determine the radii to cut the Voronoi diagram), the element column becomes a crucial section that is absent in the file you have provided.

**What can PACKMAN developers do about this situation?**
Unfortunately, nothing since we can not change the file reader to accommodate the unspecified and non-standard format.

**What can be done about this situation?**
You can add the 'element' field manually/ using another program telling PACKMAN what atom is present in the particular location (Refer to any PDB file on the RCSB database)

I hope this response helps, please contact me if you have further questions. In case the response is not noted for 3 weeks, I will close this issue. However, feel free to reopen/ contact me by email.

Thanks,
Pranav

**Comment by njbooher** [comment]
I no longer work at ISU.

**Reply by Pranavkhade** [reply]
I didn't know. Sorry about that

**Comment by cgseitz** [comment]
Hi Pranav,

Thanks for your response. I added elements to my pdb file and I still see the same issue. Is there something else I need to change about my file? I have attached it here.

Best,
Christian
[packman_glycosylated.zip](https://github.com/Pranavkhade/PACKMAN/files/11152659/packman_glycosylated.zip)

**Comment by Pranavkhade** [comment]
Hi,

I see you have added the new elements with **\t (Tab)** at the end. However, PDB file format is space separated (https://www.wwpdb.org/documentation/file-format). I have restructured your file that is attached to this email. Check if that works well for you now. I am converting this issue to a discussion.

Pranav
[re_exported.zip](https://github.com/Pranavkhade/PACKMAN/files/11154640/re_exported.zip)

**Comment by cgseitz** [comment]
Hi Pranav,

That was a silly error, thanks for catching that. It works now!

Best,
Christian
